distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response_Body.enso

```diff
@@ -18,10 +19,31 @@ type Response_Body
        - bytes: The body of the response as binary data.
     Value bytes
 
+    ## ALIAS Parse as Text
+       Decodes the body to a Text value.
+    @encoding Encoding.default_widget
+    decode_as_text : Encoding -> Text
+    decode_as_text self encoding=Encoding.utf_8 =
+        Text.from_bytes self.bytes encoding
+
+    ## ALIAS Parse as JSON, Parse JSON
+       Decodes the body as JSON.
+
+       > Example
+         Convert a response from JSON.
+
+             import Standard.Examples
+
+             example_to_text = Examples.get_geo_data.decode_as_json
+    @encoding Encoding.default_widget
+    decode_as_json : Encoding -> JS_Object | Boolean | Number | Nothing | Text | Vector
+    decode_as_json self encoding=Encoding.utf_8 =
+        self.decode_as_text encoding . parse_json
+
     ## PRIVATE
        Convert response body to Text.
     to_text : Text
-    to_text self = Text.from_utf_8 self.bytes
+    to_text self = self.decode_as_text Encoding.utf_8
 
```

Comment:
Tbh I don't like that `to_text` may fail.

This is actually problematic, because `to_text` is the delegate called when this object is passed to Java polyglot and `toString` is called on it.

When it returns a value with warnings or a dataflow error, I'm not sure what happens but I expect weird stuff may.

IMO we should just always use `decode_` to do decoding and this should be a short representation like `Response_Body ...`. Alternatively, may be worth double checking what happens if `to_text` returns warnings/dataflow-error and `toString` is called thru polyglot - I may have a test tomorrow. Maybe we should check for errors/warnings and then replace with some message "Response_Body that cannot be decoded in UTF-8, use decode_as_text with other encoding to get the contents"?